include/mbedtls/config.h

```diff
@@ -502,16 +507,19 @@
  * function, but will use your mbedtls_internal_ecp_double_jac if the group is
  * supported (your mbedtls_internal_ecp_grp_capable function returns 1 when
  * receives it as an argument). If the group is not supported then the original
- * implementation is used. The other functions and the definition of
- * mbedtls_ecp_group and mbedtls_ecp_point will not change, so your
- * implementation of mbedtls_internal_ecp_double_jac and
- * mbedtls_internal_ecp_grp_capable must be compatible with this definition.
+ * implementation is used, unless disabled by MBEDTLS_ECP_NO_FALLBACK. The
```

Comment:
Minor: I think this should be `#MBEDTLS_ECP_NO_FALLBACK`. (This is necessary for Doxygen to link to the macro definition. But sometimes you must not use `#` because the other macro is not defined in certain configurations. I'm not sure offhand whether this is a problem here: when generating useful documentation of `config.h`, all symbols need to be enabled, which `scripts/apidoc_full.sh` does.)

Reply:
Isn't this preexisting for *all* of the config.h doxygen?

Reply:
Some of it. I tend to put `#` before macro names and take it out if the CI complains.